frontend/src/components/common/csv-upload/CsvUpload.tsx

```diff
@@ -27,18 +27,22 @@ const CsvUpload = ({
       return (
         <>
           {numRecipients > 0 && (
-            <DetailBlock>
```

Comment:
this bit is changing the csv upload portion to have a summary panel, as per the image
![Screenshot 2023-07-12 at 11 07 37 AM](https://github.com/opengovsg/postmangovsg/assets/74128885/cf5a56e9-6623-423e-90b4-2b596b790d2c)